.gitignore

```diff
@@ -1 +1,2 @@
 en/_build
+/.idea
```

Comment:
this should be ignored in your globale `.gitignore` file because it is linked to your own woking environment